src/rbac/DefaultRoleManager.lua

```diff
@@ -106,11 +150,30 @@ end
 
 --       * hasLink determines whether role: name1 inherits role: name2. domain is a prefix to the roles.
 function DefaultRoleManager:hasLink(name1, name2, ...)
```

Comment:
I thought that you could refer https://github.com/casbin/casbin/blob/master/rbac/default-role-manager/role_manager.go#L173